main/zoe/api/zoe-contract-facet.md

```diff
@@ -326,6 +339,9 @@ Call when:
 - You want nothing more to happen in the contract, and
 - You don't want to take any more offers
 
+The `completion` argument is usually a string, but this 
+is not required. **tyg todo: What else can it be and what is it used for/does it do?**
```

Comment:
As above, it is used for the notification sent to the `done()` function for the contract instance. Any seats that are still open or other promises outstanding are closed with a generic 'vat terminated' message.